ner_v1/chatbot/entity_detection.py

```diff
@@ -519,7 +526,7 @@ def get_city(message, entity_name, structured_value, fallback_value, bot_message
 
 
 def get_person_name(message, entity_name, structured_value, fallback_value, bot_message,
-                    language=ENGLISH_LANG):
+                    language=ENGLISH_LANG, **kwargs):
```

Comment:
make `free_text_detection_results` as params instead of assuming it in kwargs

Reply:
Done